tools/cldr-apps/src/main/webapp/surveytool.css

```diff
@@ -1686,10 +1690,6 @@ table#voteinfo thead th:first-child {
 					sans-serif;
 }
 
-/* TODO: should load this locally via webpack, see https://unicode-org.atlassian.net/browse/CLDR-9195 */
-@import url("//fonts.googleapis.com/css?family=Noto+Sans:400italic,700italic,400,700");
-@import url("//fonts.googleapis.com/css?family=Noto+Sans+Symbols:400italic,700italic,400,700");
-
```

Comment:
The ancient ticket remains open and assigned to John: https://unicode-org.atlassian.net/browse/CLDR-9195

This change doesn't fix it but it does fix Chrome warnings that import statements need to be at the top of css